<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-PE0VDKC" failures="4" tests="37" name="testCases.VerifyLoginPageWithMultipleCredentils" time="223.375" errors="1" timestamp="2023-12-30T09:58:29 IST" skipped="11">
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="12.698"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="7.585"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="7.564"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="7.207"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="7.587">
    <skipped/>
  </testcase> <!-- LoginToApplicationWithMultiCred -->
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="7.566"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="7.387"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="7.439">
    <skipped/>
  </testcase> <!-- LoginToApplicationWithMultiCred -->
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="7.124">
    <skipped/>
  </testcase> <!-- LoginToApplicationWithMultiCred -->
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="8.586">
    <skipped/>
  </testcase> <!-- LoginToApplicationWithMultiCred -->
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="7.461">
    <skipped/>
  </testcase> <!-- LoginToApplicationWithMultiCred -->
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="12.925"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="7.594"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="8.701">
    <skipped/>
  </testcase> <!-- LoginToApplicationWithMultiCred -->
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="7.539"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="8.082"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="7.389"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="setup" time="10.619">
    <error message="no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=120.0.6099.130)
Build info: version: &#039;4.8.3&#039;, revision: &#039;e5e76298c3&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;17.0.6&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [9ed810f4316e2f433baa9c5d347ad2ca, deleteAllCookies {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 120.0.6099.130, chrome: {chromedriverVersion: 120.0.6099.109 (3419140ab66..., userDataDir: C:\Users\Admin\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:54434}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:54434/devtoo..., se:cdpVersion: 120.0.6099.130, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 9ed810f4316e2f433baa9c5d347ad2ca" type="org.openqa.selenium.NoSuchWindowException">
      <![CDATA[org.openqa.selenium.NoSuchWindowException: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=120.0.6099.130)
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.6'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [9ed810f4316e2f433baa9c5d347ad2ca, deleteAllCookies {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 120.0.6099.130, chrome: {chromedriverVersion: 120.0.6099.109 (3419140ab66..., userDataDir: C:\Users\Admin\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:54434}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:54434/devtoo..., se:cdpVersion: 120.0.6099.130, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 9ed810f4316e2f433baa9c5d347ad2ca
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:193)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:183)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:158)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:598)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:602)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteWebDriverOptions.deleteAllCookies(RemoteWebDriver.java:784)
at base.TestBase.initialization(TestBase.java:39)
at testCases.VerifyLoginPageWithMultipleCredentils.setup(VerifyLoginPageWithMultipleCredentils.java:24)
at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:823)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:590)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setup -->
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="7.569">
    <failure message="The following asserts failed:
	expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at testCases.VerifyLoginPageWithMultipleCredentils.LoginToApplicationWithMultiCred(VerifyLoginPageWithMultipleCredentils.java:52)
at jdk.internal.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- LoginToApplicationWithMultiCred -->
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="7.817"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="12.619"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="7.134">
    <skipped/>
  </testcase> <!-- LoginToApplicationWithMultiCred -->
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="7.561"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="8.053">
    <failure message="The following asserts failed:
	expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at testCases.VerifyLoginPageWithMultipleCredentils.LoginToApplicationWithMultiCred(VerifyLoginPageWithMultipleCredentils.java:52)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- LoginToApplicationWithMultiCred -->
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="12.804">
    <skipped/>
  </testcase> <!-- LoginToApplicationWithMultiCred -->
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="0.000">
    <skipped/>
  </testcase> <!-- LoginToApplicationWithMultiCred -->
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="0.000">
    <skipped/>
  </testcase> <!-- LoginToApplicationWithMultiCred -->
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="0.316">
    <failure message="The following asserts failed:
	expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at testCases.VerifyLoginPageWithMultipleCredentils.LoginToApplicationWithMultiCred(VerifyLoginPageWithMultipleCredentils.java:52)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- LoginToApplicationWithMultiCred -->
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="1.004"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="0.388"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="0.337">
    <failure message="The following asserts failed:
	expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at testCases.VerifyLoginPageWithMultipleCredentils.LoginToApplicationWithMultiCred(VerifyLoginPageWithMultipleCredentils.java:52)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- LoginToApplicationWithMultiCred -->
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="0.327"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="0.000">
    <skipped/>
  </testcase> <!-- LoginToApplicationWithMultiCred -->
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="0.322"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="0.349"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="5.373"/>
  <system-out/>
  <testcase classname="testCases.VerifyLoginPageWithMultipleCredentils" name="LoginToApplicationWithMultiCred" time="0.349"/>
  <system-out/>
</testsuite> <!-- testCases.VerifyLoginPageWithMultipleCredentils -->
